Dockerfile

```
@@ -0,0 +1,6 @@
+FROM php:7
```

```suggestion
FROM php:7.0
```
We shall ensure the library runs on the lowest PHP version supported